src/olympia/addons/indexers.py

```diff
@@ -443,7 +443,7 @@ def get_mapping(cls):
                         'id': {'type': 'long'},
                         'name': {'type': 'text'},
                         'username': {'type': 'keyword'},
-                        'is_public': {'type': 'boolean', 'index': False},
+                        'has_full_profile': {'type': 'boolean', 'index': False},
```

Comment:
I would drop it entirely from the index and from `fake_object()` in `ESAddonSerializer`. We don't need that data to build the `authors` from ES data since that is using `BaseUserSerializer`, which no longer cares about whether we're dealing with a public profile or not (and I would argue since it's for listed authors it probably didn't make sense before)

Reply:
https://github.com/mozilla/addons-server/pull/23154/commits/491bb38923e457b1ca65b2dcc681c1f75bab81f7